src/client/src/commons/form/profile/components/profileLayers/profileLayers.js

```diff
@@ -129,74 +139,96 @@ const ProfileLayers = props => {
-          {layers !== null && layers?.data?.length !== 0 && (
-            <ProfileLayersValidation
-              data={{
-                layers,
-                isEditable,
-                onUpdated,
-                selectedLayer,
-                showDelete,
-                setShowDelete,
-                selectedStratigraphyID,
-                setSelectedLayer: setSelectedLayerFunc,
-              }}
-            />
-          )}
-        </Stack>
-        {selectedLayer === null &&
-          stratigraphyKind === profileKind.STRATIGRAPHY && (
-            <Stack direction="column">
-              <Stack
-                direction="row"
-                sx={{
-                  marginBottom: 1,
-                  marginRight: 0.5,
-                  marginLeft: 0.5,
-                }}>
-                <Typography>{t("lithological_description")}</Typography>
-                {isEditable && selectedStratigraphyID !== null && (
-                  <Tooltip title={t("add")} sx={{}}>
-                    <AddCircleIcon
-                      sx={{ marginLeft: 1.5 }}
-                      data-cy="add-litho-desc-icon"
-                      onClick={addLithologicalDesc}
+      <TableContainer
+        sx={{
+          minHeight: "10em",
+          overflow: selectedLayer ? "hidden" : "",
+          borderBottom: layers?.data?.length > 0 ? "1px solid lightgrey" : "",
+        }}>
+        <Table stickyHeader aria-label="sticky table">
+          <TableHead>
+            <TableRow>
+              <TableCell>
+                <Stack direction="row">
+                  {getColumnTitle(stratigraphyKind)}
+                  {isEditable && selectedStratigraphyID !== null && (
+                    <Tooltip title={t("add")}>
+                      <AddCircleIcon
+                        sx={{ marginLeft: 1.5 }}
+                        data-cy="add-layer-icon"
+                        onClick={createNewLayer}
+                      />
+                    </Tooltip>
+                  )}
+                </Stack>
+              </TableCell>
+              {selectedLayer === null &&
+                stratigraphyKind === profileKind.STRATIGRAPHY && (
+                  <TableCell>
+                    <Stack direction="row">
+                      <Typography>{t("lithological_description")}</Typography>
+                      {isEditable && selectedStratigraphyID !== null && (
+                        <Tooltip title={t("add")} sx={{}}>
+                          <AddCircleIcon
+                            sx={{ marginLeft: 1.5 }}
+                            data-cy="add-litho-desc-icon"
+                            onClick={addLithologicalDesc}
+                          />
+                        </Tooltip>
+                      )}
+                    </Stack>
+                  </TableCell>
+                )}
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            <TableRow>
+              <td style={cellStyle}>
```

Comment:
Q: Was ist der Grund hier nicht die MUI `TableCell` zu verwenden?

Reply:
Es hat was mit dem Serverside rendering des Mui Tables zu tun. Die Komponenten werden dann auf dem Dom "hydrated". 
Irgendwie gabs damit ein Problem, jedenfalls wurde alles innerhalb der Zelle bei jedem render neu gemounted und der State ging verloren.  Ist jetzt nicht so schön, vermutlich hätte man am besten von vorneherein einen Table genommen und dann das mapping der einzelnen "Lithological-Descriptions" in den Zellen gemacht, statt eine komplette gemappte Liste in eine einzelne Zelle zu packen. 
Ich würde das nochmal anschauen, sobald ich die Facies-Description hinzufüge.  Ggf. ist dann die Umstellung sinnvoll.